.travis.yml

```diff
@@ -5,23 +5,34 @@ os:
   - macos
   - linux
 elixir:
-  - 1.6
-  - 1.7
-  - 1.8
-  - 1.9
+  - '1.6'
+  - '1.7'
+  - '1.8'
+  - '1.9'
+  - '1.10'
 otp_release:
-  - 19.3
-  - 20.3
-  - 21.0
+  - '19.3'
+  - '20.3'
+  - '21.0'
+  - '22.2'
 matrix:
   exclude:
     - os: windows
-    - elixir: 1.8
-      otp_release: 19.3
-    - elixir: 1.9
-      otp_release: 19.3
+    - elixir: '1.6'
+      otp_release: '22.2'
+    - elixir: '1.7'
+      otp_release: '22.2'
```

Comment:
These elixir / OTP groups failed in CI, so are disabled:

* 1.6 / 22
* 1.7 / 22